doc/src/manual/mathematical-operations.md

```diff
@@ -394,7 +394,7 @@ Julia applies the following order and associativity of operations, from highest
 |:-------------- |:------------------------------------------------------------------------------------------------- |:-------------------------- |
 | Syntax         | `.` followed by `::`                                                                              | Left                       |
 | Exponentiation | `^`                                                                                               | Right                      |
-| Unary          | `+ - √`                                                                                           | Right[^1]                  |
+| Unary          | `+ - ! √`                                                                                         | Right[^1]                  |
```

Comment:
```suggestion
| Unary          | `+ - ! ~ ¬ √ ∛ ∜ ⋆ ± ∓ <: >:`                                                                     | Right[^1]                  |
```

According to the link below ("For a complete list of *every* Julia operator's precedence") this is the complete list (aside from the dotted versions of them). Should we go with just adding all of them to this? @JeffBezanson